project_templates/roundtable_aiohttp_bot/example/.github/workflows/build.yaml

```diff
@@ -0,0 +1,48 @@
+name: Docker
```

Comment:
Do you mean to delete this example file? (Unfortunately, the CI routine doesn't delete examples and then doesn't check for files that shouldn't exist).